documentation/modules/exploit/windows/fileformat/device_viewer.md

```diff
@@ -0,0 +1,115 @@
+## Vulnerable Application
+
+This module exploits a SEH stack buffer overflow in DeviceViewer version 3.10.12.0. By creating a specially crafted file and copying its content in the "User" login field, an attacker will be able to gain arbitrary code execution.
+
+Link to vulnerable software [EDB](https://www.exploit-db.com/apps/4d10486a079bd1f1864c30e86cd2aa80-DeviceViewer.exe) - [VoidSec](https://github.com/VoidSec/Exploit-Development/blob/master/windows/x86/local/DeviceViewer_v.3.10.12.0/DeviceViewer.exe)
```

Comment:
Thanks for including these links, and also saving it locally to your repo, this is very helpful for testers!